packages/core/src/filters/FilterSystem.js

```diff
@@ -330,8 +331,8 @@ export default class FilterSystem extends System
      * Draws a filter.
      *
      * @param {PIXI.Filter} filter - The filter to draw.
-     * @param {PIXI.RenderTarget} input - The input render target.
-     * @param {PIXI.RenderTarget} output - The target to output to.
+     * @param {PIXI.RenderTexture} input - The input render target.
+     * @param {PIXI.RenderTexture} output - The target to output to.
```

Comment:
Are these RenderTarget -> RenderTexture changes correct? In the past these have been different things.

Reply:
RenderTexture/RenderTarget functionality was moved to RenderTexture/Framebuffer